here-naksha-app-service/src/test/java/com/here/naksha/app/service/ViewTypeUnionAllTest.java

```diff
@@ -0,0 +1,79 @@
+package com.here.naksha.app.service;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+import com.github.tomakehurst.wiremock.matching.UrlPattern;
+import com.here.naksha.app.common.ApiTest;
+import com.here.naksha.app.common.NakshaTestWebClient;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpResponse;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.here.naksha.app.common.CommonApiTestSetup.setupSpaceAndRelatedResources;
+import static com.here.naksha.app.common.TestUtil.loadFileOrFail;
+import static com.here.naksha.app.common.assertions.ResponseAssertions.assertThat;
+
+@WireMockTest(httpPort = 9090)
+public class ViewTypeUnionAllTest extends ApiTest {
```

Comment:
sorry, but I think it is better we replace unionall everywhere (like file, folder name as well) otherwise in future when we want to implement unionall then it will create confusion. agree?

Reply:
yes, sorry for not noticing, pushed the fix